pkg/sysregistriesv2/alias.go

```diff
@@ -0,0 +1,112 @@
+package sysregistriesv2
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/storage/pkg/homedir"
+	"github.com/containers/storage/pkg/lockfile"
+)
+
+// userAliasesFile is the user-specific registries.conf file to store aliases.
+var userAliasesFile = filepath.FromSlash(".config/containers/aliases.conf")
```

Comment:
`short-name-aliases.conf`? Users hopefully don’t need to type this, and many things in the container world could be or have aliases.

Reply:
Good point. I see wisdom in making that explicit in files that user will easily hit.